packages/site/src/layout/PageWrapper.tsx

```diff
@@ -7,13 +7,14 @@ import { PropsWithChildren } from "react";
  */
 export const PageWrapper = ({ children }: PropsWithChildren) => {
   return (
-    <main
+    <div
       style={{
         width: "100%",
         minHeight: "100%",
+        backgroundColor: "var(--color-surface)",
```

Comment:
This relates to the change [here](https://github.com/GetJobber/atlantis/pull/2246/files#r1884652961)